backend/oasst_backend/utils/hugging_face.py

```diff
@@ -1,10 +1,20 @@
+from enum import Enum
 from typing import Any, Dict
 
 import aiohttp
 from oasst_backend.config import settings
 from oasst_shared.exceptions import OasstError, OasstErrorCode
 
 
+class HF_url(str, Enum):
+    HUGGINGFACE_TOXIC_ROBERTA = ("https://api-inference.huggingface.co/models/unitary/multilingual-toxic-xlm-roberta",)
+    HUGGINGFACE_FEATURE_EXTRACTION = "https://api-inference.huggingface.co/pipeline/feature-extraction"
+
+
+class HF_embeddingModel(str, Enum):
+    MINILM = "sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2"
```

Comment:
just "MINILM" is probably a bit too scarce, given that there are many different minilms. I guess it can stay for now, but we should keep this in mind for the past